chapters/offenses.adoc

```diff
@@ -40,6 +48,8 @@ NOTE: Dribblers can still be used to dribble large distances with the ball as lo
 ==== Ball Speed
 A robot must not accelerate the ball faster than 6.5 meters per second in 3D space.
 
+Violations do not stop game play. Goals scored with kicks exceeding the ball speed limit will not be counted. Increase <<Fouls, foul counter>> by one.
```

Comment:
We should use the grace period here as well. Making a fast pass and than scoring a goal should also not be allowed.